.github/workflows/upgrade-test.yml

```diff
@@ -0,0 +1,157 @@
+name: DSP Install and Upgrade
+
+on:
+  workflow_dispatch:
+    inputs:
+      released-version:
+        default: 'v1.y.y'
+        description: 'Most recently released DSP version'
+        required: true
+      candidate-version:
+        default: 'v1.x.x'
+        description: 'Candidate version for upgrade'
+        required: true
+
+concurrency:
+  group: ${{ github.head_ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  kubernetes-olm-upgrade:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 60
+    env:
+      OLM_VERSION: v0.24.0
+      VERSION: "v0.0.0-ghaction"  # Need to supply some semver version for bundle to be properly generated
+      CATALOG_BASE_IMG: "registry.access.redhat.com/redhat/community-operator-index:v4.13"
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0  # fetching also previous commits to get tags
```

Comment:
I guess you don't need this.
I used the complete fetch to be able to retrieve released version based on tag in GitHub repo.

Reply:
I believe this step would be useful for us, in order to retrieve all history of the repo, keeping it for now.